spec/sum.js

```diff
@@ -0,0 +1,4 @@
+function sum(a, b) {
+  return a + b;
+}
+module.exports = sum;
```

Comment:
Nice work verifying that Jest was working! No need to ship this to production though. Delete this file.